sdks/python/test-suites/tox/py3/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Unit tests for Python 3
+ */
+
+apply plugin: org.apache.beam.gradle.BeamModulePlugin
+applyPythonNature()
+
+// Required to setup a Python 3 virtualenv.
+project.ext.python3 = true
+
+task lint {}
+check.dependsOn lint
+
+toxTask "lintPy3", "py3-lint"
+lint.dependsOn lintPy3
+
+toxTask "testPython3", "py3"
+test.dependsOn testPython3
+
+toxTask "testPy3Gcp", "py3-gcp"
+test.dependsOn testPy3Gcp
+
+toxTask "testPy3Cython", "py3-cython"
+test.dependsOn testPy3Cython
+// Ensure that testPy3Cython runs exclusively to other tests.
+testPy3Cython.mustRunAfter testPython3, testPy3Gcp
+testPy3Cython.mustRunAfter ':beam-sdks-python:testPy2Cython'
```

Comment:
Why is this rule needed?

Reply:
testPy3Cython and testPy2Cython run in parallel even I use `finalizedBy` in [here](https://github.com/apache/beam/pull/7675/files#diff-c197962302397baf3a4cc36463dce5eaR197). If they do not affect each other, I can remove this line.